src/configs/core.js

```diff
@@ -51,6 +51,7 @@ exports.plugins = [
   'redisCluster',
   'http',
   'prometheus',
+  'consul',
```

Comment:
this is a breaking change, instead of updating default list of plugins, setup this as a dependency on the cf-worker and its action. if worker or action is enabled during startup, push consul plugin into init configuration

Reply:
Added check whether `consul` plugin included in configuration, for now.
Think we should add a feature that allows us to include plugins in the `constructor` of the `@microfleet/core` plugin system?